openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -196,6 +196,24 @@ send_data() {
 	done
 }
 
+register() {
+	logger -s "Waiting for device to register" \
+		-t openwisp-monitoring \
+		-p daemon.info
+	# shellcheck disable=SC2002
+	{
+		UUID=$(cmd /etc/config/openwisp | awk '/option uuid / { print $3 }' | tr -d "'")
+		KEY=$(cat /etc/config/openwisp | awk '/option key / { print $3 }' | tr -d "'")
+	}
```

Comment:
I think it's not good to ignore shellcheck recommendations
```suggestion
	UUID=$(awk '/option uuid / { print $3 }' < /etc/config/openwisp | tr -d "'")
	KEY=$(awk '/option key / { print $3 }' < /etc/config/openwisp | tr -d "'")
```

Reply:
@codesankalp please use `uci get` here and remove the `shellcheck disable=SC2002`, consult the `uci` command docs on openwrt https://openwrt.org/docs/guide-user/base-system/uci or look at how it is used in openwisp-config.